knox.go

```diff
@@ -1,12 +1,21 @@
 package knox
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
+	"time"
 
 	"github.com/knox-networks/knox-go/credential_adapter"
+	"github.com/knox-networks/knox-go/model"
+	mb "github.com/multiformats/go-multibase"
+	"github.com/piprate/json-gold/ld"
 )
 
+const NORMALIZATION_ALGO = "URDNA2015"
```

Comment:
Correct me if i'm wrong but i didn't see any special code convention for constant, I think it can be like NormalizationAlgo

Reply:
Resolving this duplicate comment